lib/identity_cache/query_api.rb

```diff
@@ -450,6 +450,24 @@ def prefetch_one_association(association, records)
         end
         next_level_records
       end
+
+      def expire_primary_index(id) # :nodoc:
+        return unless primary_cache_index_enabled
+
+        IdentityCache.logger.debug do
+          extra_keys =
+            if respond_to?(:updated_at)
+              old_updated_at = old_values_for_fields([:updated_at]).first
+              "expiring_last_updated_at=#{old_updated_at}"
+            else
+              ""
+            end
+
+          "[IdentityCache] expiring=#{name} expiring_id=#{id} #{extra_keys}"
+        end
```

Comment:
This debugging code depends on more than just the `id` from the instance.  I think we can just remove it.  I didn't even notice this code before today since we don't usually use the debug log level.